hw10_program_optimization/stats.go

```diff
@@ -19,8 +23,10 @@ type User struct {
 	Address  string
 }
 
+type users [100_000]User
 type DomainStat map[string]int
 
+// nolint: errorlint
```

Comment:
Зачем?

Рекомендую оставлять поясняющий комментарий. Например — https://github.com/kulti/task-list/blob/master/server/internal/models/task_state_test.go#L38.

О nolint рекомендую прочитать тут — https://golangci-lint.run/usage/false-positives/#nolint). В частности, обратить внимание на:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

Reply:
Линтер ругался, когда в Errorf() плейсхолдер ошибки "%v", а не "%w". Спасибо за замечания